data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,469 @@ mission "Coalition: Submarines 4"
+			`	The recording ends, and you two stay in an awkward silence for a minute. "Aware that he had died, I was. Too long it has been. Even if found a safe haven, habitable planet or a friendly group, the fleet had, I knew that long dead, all of them would be." She pauses, and the silence returns for another minute. She shuts off the pylon. "Only, expecting something... anything, I was. A heroic death, a summary of findings. Accomplished nothing, they did. Right after all, the consuls were."`
+			choice
+				`	"Heroic death or not, he still got some information, didn't he? Information you recovered."`
+					goto info
+				`	"He didn't get the chance to act heroically, he was ambushed."`
+			`	"Massacred, if truthful about the Quarg's numbers, he was," she responds. "Risked a great deal of jump drives, the sixth expedition had. Thought, it was, that with a greater number of Punishers, a greater chance of a safe return, there was." She sighs. "Just decrease our reserve of them, it did."`
+				goto back
+			label info
+			`	"Inconclusive reports on red nebulas or disappearing ruins, what good are they?" she sighs. "After met up in their pincer movement, the groups had, after exploring points of interest, supposed to drive further than we had previously, they were. To reach the galactic core, to maybe find friendly groups to facilitate more trips... achieved, none of the goals were."`
+			label back
+			`	She goes back to her bunk, saying she wishes to be alone for a while, and you fly to the planet's hangars. A few hours later, she comes back out. "Finished my report, I have. Ready to go back home I am, Captain <last>. A long trip this was, and promised to pay you well, I did, so when <planet> we reach, give you your well earned 10,000,000 credits, I shall. Present my findings to the council, I must."`
+			choice
+				`	"That's a lot of money. I was worried you wouldn't even want to pay me, with how disappointed you were earlier."`
+					goto alot
```

Comment:
I get the label isn't in dialogue so spellcheck isn't especially relevant, but let's go ahead and change it so all your tests are passing.